src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 * Fix the issue of incorrect handling of null attributes.
   ([#566](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/566))
+* Support for Google Cloud Dependencies up to 4.0.0
```

Comment:
The package reference is added to support up to 3.x , but not 4.0 and higher.

https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/dependencies#nuget-dependency-version-ranges